index.html

```diff
@@ -7,227 +7,278 @@
   <link rel="stylesheet" href="quiz.css">
 </head>
 <body>
-  <article id="Widgets">
-    <div id="cards">
+  <div id="cards">
 
-      <div class="card" id="widget-A">
-        <div class="card-header-block">
-          <header class="card-header">
-            Widget A
-          </header>
-          <br>
+    <article class="card" id="widget-A">
+
+      <div class="card-header-block">
+        <header class="card-header">
+          <h1>Widget A</h1>
+        </header>
+        <br>
```

Comment:
You can achieve the same effects you are wanting by using the `<br>`s with padding and margins in your CSS.